firefox-ios/firefox-ios-tests/Tests/L10nSnapshotTests.xctestplan

```diff
@@ -0,0 +1,32 @@
+{
+  "configurations" : [
+    {
+      "id" : "E8531BAA-6B58-46CF-827B-7A30FEF28AD0",
+      "name" : "Configuration 1",
+      "options" : {
+
+      }
+    }
+  ],
+  "defaultOptions" : {
+    "testRepetitionMode" : "retryOnFailure",
```

Comment:
I have enabled "Retry on Failure" for the test plan's config. The default value is 3.